orders/src/main/java/local/naught/orders/model/Customers.java

```diff
@@ -0,0 +1,209 @@
+package local.naught.orders.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "customers")
+public class Customers
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long custcode;
+
+    @Column(nullable = false)
+    private String custname;
+
+    private String custcity;
+    private String workingarea;
+    private String custcountry;
+    private String grade;
+    private double openingamt;
+    private double receiveamt;
+    private double paymentamt;
+    private double outstandingamt;
+    private String phone;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "agentcode",
+            nullable = false)
+    @JsonIgnoreProperties({"customers", "hibernateLazyInitializer"})
+    private Agents agent;
+
+    @OneToMany(mappedBy = "customer",
+            cascade = CascadeType.ALL,
+            orphanRemoval = true)
+    @JsonIgnoreProperties({"customers", "hibernateLazyInitializer"})
+    private List<Orders> orders = new ArrayList<>();
```

Comment:
Line 39 - Use an array of strings, and make sure to both correct the property typo (singular **"customer"**, not plural "customers") and include the property "agent" as well to avoid repeated printing of properties.

Also, you won't have to ignore `hibernateLazyInitializer` - it won't show up anyway since you're not having lazy fetching in a one-to-many relationship.

To use the array of strings: **`@JsonIgnoreProperties({"customer", "agent"})`**